Fix - Update SDK docs (#1047)
iotaledger/iota-wiki: next/docs/iota-sdk/docs/troubleshooting.md

# Troubleshooting

When you encounter a problem or need assistance while working with IOTA, there are several resources available to you.

## IOTA StackExchange

The [IOTA StackExchange](https://iota.stackexchange.com/) is a valuable tool for developers seeking answers to their
questions. By simply searching for your specific problem, you can find existing solutions and insights from the IOTA
community.

## Discord Community

> If you cannot find an immediate solution to your question, you can post your query on the dedicated IOTA-SDK channel
> within the official [IOTA Discord](https://discord.iota.org) server. By sharing your question with the community, you
> can engage with fellow developers and experts who can provide guidance and assistance.

Reviewer: ```suggestion
If you cannot find an immediate solution to your question, you may find help in the dedicated IOTA-SDK channel
within the official [IOTA Discord server](https://discord.iota.org). There you can engage with fellow developers and experts who can provide guidance and assistance.
```

## Joining the Discussion

If you wish to participate in broader discussions about IOTA SDK or seek additional support, you can visit the

> #clients-discussion channel on the [IOTA Discord](https://discord.iota.org). This channel serves as a hub for

Reviewer 2: ```suggestion
#development channel on the [IOTA Discord](https://discord.iota.org). This channel serves as a hub for
```

Reviewer 2: Or #client-libs, #wallet-libs

> If you wish to participate in broader discussions about IOTA SDK or seek additional support, you can visit the
> #development channel on the [IOTA Discord](https://discord.iota.org). This channel serves as a hub for
> discussions related to the IOTA client libraries, where you can interact with other developers and receive further
> assistance.

Reviewer: ```suggestion
If you wish to participate in broader discussions about the IOTA SDK or seek additional support, you can visit the
`#development` channel on the [IOTA Discord server](https://discord.iota.org). This channel serves as a hub for
discussions related to the IOTA client libraries, where you may interact with other developers and receive further
assistance.
```
